src/sentry/identity/base.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import absolute_import, print_function
+
+import logging
+from collections import namedtuple
+
+from sentry.utils.pipeline import PipelineProvider
+
+
+class MigratingIdentityId(namedtuple('MigratingIdentityId', ['id', 'legacy_id'])):
```

Comment:
Yeah, I actually introduced this over here https://github.com/getsentry/sentry/pull/6577, but am moving it into the identity module, and it will be removed from the auth provider later once the auth providers make use of the identity provider.